src/multibody/model.hpp

```diff
@@ -124,6 +124,9 @@ namespace se3
       , gravity( gravity981,Eigen::Vector3d::Zero() )
     {
       names[0]     = "universe";     // Should be "universe joint (trivial)"
+      // FIXME Should the universe joint be a FIXED_JOINT even if it is
+      // in the list of joints ? See comment in definition of
+      // Model::addJointFrame and Model::addBodyFrame
       addFrame(Frame("universe", 0, 0, SE3::Identity(), FIXED_JOINT));
```

Comment:
Naively, I added frame "universe" with type FIXED_JOINT.

I realize now it can also be considered a JOINT because it is in `model.joints`. Any opinion ?